src/mongo/db/pipeline/document_source_group.cpp

```diff
@@ -494,7 +491,7 @@ DocumentSource::GetNextResult DocumentSourceGroup::initialize() {
         return DocumentSource::GetNextResult::makeEOF();
     }
 
-    dassert(numAccumulators == vpExpression.size());
+    dassert(numAccumulators == _accumulatedFields.size());
```

Comment:
This doesn't do anything useful any more! We know these are equal! You declared it as such on line 465.